stable/prometheus-node-exporter/templates/daemonset.yaml

```diff
@@ -2,6 +2,7 @@ apiVersion: apps/v1
 kind: DaemonSet
 metadata:
   name: {{ template "prometheus-node-exporter.fullname" . }}
+  namespace: {{ .Release.Namespace }}
```

Comment:
Please replace `{{ .Release.Namespace }}` with `{{ $.Release.Namespace }}` This is less error prone, especially when folks start copy-pasting it.